examples/qml_extension_plugin/tests/CMakeLists.txt

```diff
@@ -4,23 +4,29 @@
 #
 # SPDX-License-Identifier: MIT OR Apache-2.0
```

Comment:
can you try removing `example_qml_extension_plugin_test|example_qml_features_test|example_qml_minimal_myobject_test` and the comment "qmltestrunner fails to import QtQuick module" from `.github/workflows/github-cxx-qt-tests.yml` so we can see if CI likes it?

Reply:
Am currently doing that in a separate commit.
Would like to get this working on Linux CI first, then see if it fixes #110